styles/_fieldset.scss

```diff
@@ -0,0 +1,73 @@
+@layer reset, default, disabled, focus;
+:root {
```

Comment:
Can you use 

@import "../node_modules/@cdssnc/gcds-tokens/build/web/css/components/fieldset.css";

I did that with the text input etc...

Would prefer if we can just get the vars from them vs the hardcoded.